interop_binaries/src/lib.rs

```diff
@@ -625,10 +625,10 @@ pub mod test_util {
             .timeout(Duration::from_secs(1))
             .build()
             .unwrap();
-        let url = Url::parse(&format!("http://127.0.0.1:{port}/")).unwrap();
+        let url = Url::parse(&format!("http://127.0.0.1:{port}/internal/test/ready")).unwrap();
```

Comment:
This change would duplicate the above `await_ready_ok()` function. We could instead use that in `Daphne::new()`.

Reply:
Done!